ManagerRegistry.php

```diff
@@ -20,12 +20,36 @@
 
 namespace Doctrine\Bundle\PHPCRBundle;
 
+use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Bridge\Doctrine\ManagerRegistry as BaseManagerRegistry;
 use Doctrine\ODM\PHPCR\PHPCRException;
 
 class ManagerRegistry extends BaseManagerRegistry
 {
     /**
+     * Construct.
+     *
+     * @param ContainerInterface $container
+     * @param array              $connections
+     * @param array              $entityManagers
+     * @param string             $defaultConnection
+     * @param string             $defaultEntityManager
+     * @param string             $proxyInterfaceName
+     */
+    public function __construct(
+        ContainerInterface $container,
+        array $connections,
+        array $entityManagers,
+        $defaultConnection,
+        $defaultEntityManager,
```

Comment:
we should append Name to this and the $defaultConnection and then we can remove the docblock as it will not provide any additional information.